templates/tableau-cluster-windows-master.template

```diff
@@ -512,7 +512,8 @@
                             {
                                 "Ref": "QSS3KeyPrefix"
                             },
-                            "templates/windows-bastion.template"
+                            "submodules/quickstart-microsoft-rdgateway",
+							"templates/rdgw-standalone.template"
```

Comment:
Did you review the parameters being passed for this? I don't think they cover the requirements of the template. See this as an example for more details: https://github.com/aws-quickstart/quickstart-tableau-awsdata/blob/master/templates/tableau-awsdata-master.template

Reply:
Minor comment and not a necessary change: Use a Sub instead of Join :)

Reply:
sorry, I didn't review the parameters being passed for tableau-awsdata-master.template.

Reply:
here is input parameters for rdgw-standalone.template:
                "Parameters": {
                    "AdminPassword": {
                        "Ref": "RDGWAdminPassword"
                    },
                    "AdminUser": {
                        "Ref": "RDGWAdminUser"
                    },
                    "KeyPairName": {
                        "Ref": "KeyPairName"
                    },
                    "DomainDNSName": {
                        "Ref": "DomainDNSName"
                    },
                    "PublicSubnet1ID": {
                        "Fn::GetAtt": [
                            "VPCStack",
                            "Outputs.PublicSubnet1ID"
                        ]
                    },
                    "PublicSubnet2ID": {
                        "Fn::GetAtt": [
                            "VPCStack",
                            "Outputs.PublicSubnet2ID"
                        ]
                    },
                    "QSS3BucketName": {
                        "Ref": "QSS3BucketName"
                    },
                    "QSS3KeyPrefix": {
                        "Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-rdgateway/"
                    },
                    "RDGWInstanceType": {
                        "Ref": "RDGWInstanceType"
                    },
                    "RDGWCIDR": {
                        "Fn::If": [
                            "NoRDGWInboundCIDRCondition",
                            {
                                "Fn::GetAtt": [
                                    "VPCStack",
                                    "Outputs.VPCCIDR"
                                ]
                            },
                            {
                                "Ref": "RDGWInboundCIDR"
                            }
                        ]
                    },
                    "VPCID": {
                        "Fn::GetAtt": [
                            "VPCStack",
                            "Outputs.VPCID"
                        ]
                    }
                }
            }

we don't have input parameters AdminPassword, AdminUser and DomainDNSName on this windows template, so I don't think rdgw-standalone.template can be used here. Should I change it back to Windows-bastion?

Reply:
You can go ahead and add them. It's a best practice to not use the built-in administrator account...

Reply:
Can I do it in another pr? I don't want to change windows bastion script in this pr, but I can create a story and do it later. The purpose of this pr is to check in Linux support for tableau server cluster.

Reply:
Yeah, that's fine.